frontend/src/HelperFunctions/validateAndReturnUrl.js

```diff
@@ -0,0 +1,7 @@
+export default function validateAndReturnUrl(url) {
+	var pattern = /^((http|https):\/\/)/;
+	if(!pattern.test(url)) {
+		url = "https://" + url;
```

Comment:
This isn't actually validating that the rest of the URL is safe though. Perhaps we could pass this through `new URL` which would check the end result is a properly formed URL.

Reply:
Done